portal/index.html

```diff
@@ -0,0 +1,59 @@
+<html>
+
+    <center id="loginform" style="padding-top: 10em">
+
+        <p>
+        <label for="username"><b>Username</b></label>
+        <input id="username" type="text" placeholder="Enter Username" name="username" required>
+        </p>
+
+        <p>
+        <label for="password"><b>Password</b></label>
+        <input id="password" type="password" placeholder="Enter Password" name="password" required>
+        </p>
+
+        <button id="login">Login</button>
+    </center>
+
+    <center id="userdata" style="color:green;">
+    </center>
+
+
```

Comment:
I suggest, if it's easy, to find a way to add a "Legal mention link" here. Indeed adding a Legal mention here help to avoid Google Safe browsing flag.

The idea is to display pages that are not totally identical and without Legal mention.

Reply:
This was just a yolo page to test the new API, the actual new portal is being worked on in https://github.com/YunoHost/yunohost-portal

But indeed, we should also adjust the API and make this configurable somehow

Reply:
For this need, only a simple "Legal mention" link (with translation) that send onto a link that we can configure in the main settings ?

I don't know if a custom footer is really needed. Maybe to add GDPR specific links or return to the main site ? It's also possible that doing that decrease more the Flag score, but for the moment changing this was enough for me to be remove from Google Safebrowsing blacklist...